### PR TITLE
Added Snippets for DNS Zone, DNS Record, ACI

### PR DESCRIPTION
**Pull request by sam-cogan**
New snippets for 

- DNS Zone
- DNS Record
- Azure Container Instance

### CODE REVIEW

File: VSCode/armsnippets.json [hunk 1]
```diff
@@ -1161,4 +1161,83 @@
...
+    "description": "New DNS Record"
+},
+
+"Azure Container Instance":{
+    "prefix":"arm-container-instance",
+    "body":[
+        "{",
+            "    \"name\": \"${1:containerGroupName}\",",
+            "    \"type\": \"Microsoft.ContainerInstance/containerGroups\",",
+            "    \"apiVersion\": \"2017-08-01-preview\",",
```
Comment (philon-msft) [inline VSCode/armsnippets.json]: Newer apiVersion 2017-10-01-preview is available

File: VSCode/armsnippets.json [hunk 2]
```diff
@@ -1161,4 +1161,83 @@
...
+        "      \"${4:dnsZoneName}\"",
+        "    ],",
+        "    \"properties\": {",
+        "      \"TTL\": 3600,",
+        "      \"${5|Arecords,AAAARecords,MXRecords,NSRecords,PTRRecords,SRVRecords,TXTRecords,CNAMERecord,SOARecord|}\": [",
+        "      ]",
+        "    }",
+        " }"
+    ],
+    "description": "New DNS Record"
```
Comment (philon-msft) [inline VSCode/armsnippets.json]: No need to include "New" in descriptions

File: VSCode/armsnippets.json [hunk 3]
```diff
@@ -1161,4 +1161,83 @@
...
     ],
     "description": "OMS management solution"
+},
+"DNS Zone":{
+    "prefix":"arm-dns-zone",
+    "body":[
+        "{",
+        "    \"type\": \"Microsoft.Network/dnszones\",",
+        "    \"name\": \"${1:dnsZoneName}\",",
+        "    \"apiVersion\": \"2016-04-01\",",
```
Comment (philon-msft) [inline VSCode/armsnippets.json]: Newer apiVersion 2017-09-01 available

File: VSCode/armsnippets.json [hunk 4]
```diff
@@ -1161,4 +1161,83 @@
...
+},
+
+
+"DNS Record":{
+    "prefix":"arm-dns-record",
+    "body":[
+        "{",
+        "    \"type\": \"Microsoft.Network/dnszones/${1|A,AAAA,CNAME,MX,NS,PTR,SOA,SRV,TXT|},\",",
+        "    \"name\": \"[concat(${2:dnsZoneName}, '/', ${3:newRecordName}))]\",",
+        "    \"apiVersion\": \"2016-04-01\",",
```
Comment (philon-msft) [inline VSCode/armsnippets.json]: update apiVersion

**Comment by sam-cogan** [comment]
Updated as requested.

**Comment by philon-msft** [comment]
Thanks for the contribution, @sam-cogan!

**Pull request merged**